addons/editor-random-direction-block/addon.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Point towards random direction block",
+  "description": "Adds a \"random position\" option to the \"point towards ()\" block that makes the sprite point in a random direction. This will work for everyone using the project.",
```

Comment:
```suggestion
  "description": "Adds a \"random direction\" option to the \"point towards ()\" block that makes the sprite point in a random direction. This will work for everyone using the project.",
```